### PR TITLE
allow retrieving tags from graphql

### PR DESCRIPTION
**Pull request by rileylnapier**
## Description

pull back tags from graphql listMessages and getMessageCount

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: packages/client-graphql/__tests__/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      Object {
+        "body": "{\\"query\\":\\"query GetMessages($params: FilterParamsInput, $limit: Int = 10, $after: String) {\\\\n  messages(params: $params, limit: $limit, after: $after) {\\\\n    totalCount\\\\n    pageInfo {\\\\n      startCursor\\\\n      hasNextPage\\\\n      __typename\\\\n    }\\\\n    nodes {\\\\n      id\\\\n      messageId\\\\n      created\\\\n      read\\\\n      tags\\\\n      content {\\\\n        title\\\\n        body\\\\n        blocks {\\\\n          ... on TextBlock {\\\\n            type\\\\n            text\\\\n            __typename\\\\n          }\\\\n          ... on ActionBlock {\\\\n            type\\\\n            text\\\\n            url\\\\n            __typename\\\\n          }\\\\n          __typename\\\\n        }\\\\n        data\\\\n        trackingIds {\\\\n          clickTrackingId\\\\n          deliverTrackingId\\\\n          readTrackingId\\\\n          unreadTrackingId\\\\n          __typename\\\\n        }\\\\n        __typename\\\\n      }\\\\n      __typename\\\\n    }\\\\n    __typename\\\\n  }\\\\n}\\\\n\\",\\"operationName\\":\\"GetMessages\\",\\"variables\\":{}}",
```
Comment (rileylnapier) [inline packages/client-graphql/__tests__/index.spec.ts:22]: i wonder if there are graphql query helpers that can be used here?

File: packages/client-graphql/src/messages/index.ts [hunk 2]
```diff
@@ -2,30 +2,28 @@ import { Client } from "urql";
...
+  tags?: string[];
   from?: number;
+  isRead?: boolean;
 }
-type GetUnreadMessageCount = (params?: IMessageCountParams) => Promise<number>;
-export const getUnreadMessageCount = (
-  client?: Client
-): GetUnreadMessageCount => async (params) => {
+type GetMessageCount = (params?: IMessageCountParams) => Promise<number>;
+export const getMessageCount = (client?: Client): GetMessageCount => async (
```
Comment (rileylnapier) [inline packages/client-graphql/src/messages/index.ts:17]: nobody is importing this one directly yet so i can change it.  i verified with color

File: packages/components/src/index.html [hunk 3]
```diff
@@ -35,8 +35,9 @@
         });
       };*/
       window.courierConfig = {
-        clientKey: "Y2U3OWI3NGEtY2FhZC00NTFjLTliZDMtMGZkOTVhMmQ0ZWE4",
-        userId: "Google_108669107033656954156",
+        apiUrl: "https://rubmz24skk.execute-api.us-east-1.amazonaws.com/dev",
```
Comment (rileylnapier) [inline packages/components/src/index.html:38]: we need a better way to handle these

File: packages/client-graphql/package.json [hunk 4]
```diff
@@ -9,14 +9,15 @@
...
-    "test": "jest -c jest.config.js --runInBand --silent",
+    "test": "jest -c jest.config.js --runInBand",
     "type-check": "tsc --noEmit",
     "readme": "concat-md --toc --decrease-title-levels --dir-name-as-title docs > README.md",
     "types": "tsc --declaration --outDir typings/ --emitDeclarationOnly --declarationMap --allowJs false --checkJs false"
   },
   "license": "ISC",
   "dependencies": {
     "graphql": "^15.5.0",
+    "isomorphic-fetch": "^3.0.0",
```
Comment (tk26) [inline packages/client-graphql/package.json:20]: do we need this?

Comment (rileylnapier) [reply packages/client-graphql/package.json:20]: ill remove

**Review by tk26 — APPROVED** [review]

**Pull request merged**